<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="2" errors="0" failures="2" tests="13" time="99.664">
 <testsuite name="damostrate data driven test" timestamp="2020-06-13T12:48:10" hostname="localhost" time="13.492" errors="0" tests="3" skipped="0" disabled="0" failures="0">
  <testcase classname="damostrate data driven test" name="home page data driven using dataprividerurlCurrent_Content_FirstDataSets" time="7.308" />
  <testcase classname="damostrate data driven test" name="home page data driven using dataprividerurlCurrent_Content_SecondDataSets" time="3.359" />
  <testcase classname="damostrate data driven test" name="home page data driven using dataprividerurlCurrent_Content_ThirdDataSets" time="2.817" />
 </testsuite>
 <testsuite name="performing web interatction" timestamp="2020-06-13T12:48:23" hostname="localhost" time="32.973" errors="0" tests="2" skipped="1" disabled="0" failures="2">
  <testcase classname="performing web interatction" name="drag and drop test case" time="0">
   <skipped message="Temporarily disabled with xit" />
  </testcase>
  <testcase classname="performing web interatction" name="selectable vrification test demo" time="32.97">
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at Timeout._onTimeout (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)]]>
   </failure>
   <failure type="exception" message="Failed: TimeoutError: Element didnot found
Wait timed out after 31146ms"><![CDATA[Error: TimeoutError: Element didnot found
Wait timed out after 31146ms
    at ElementAction.<anonymous> (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\utility\ElementAction.ts:23:19)
    at Generator.throw (<anonymous>)
    at rejected (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\JSFiles\utility\ElementAction.js:6:65)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: Run it("selectable vrification test demo") in control flow
    at UserContext.<anonymous> (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.describe (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\TestStep\InteractionAction.ts:29:5)
    at addSpecsToSuite (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\Users\mrmra\.jenkins\TypeScriptJaminePOM_Projecct\TestStep\InteractionAction.ts:6:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)]]>
   </failure>
  </testcase>
 </testsuite>
 <testsuite name="performimg menu actions" timestamp="2020-06-13T12:48:56" hostname="localhost" time="0.005" errors="0" tests="2" skipped="0" disabled="2" failures="0">
  <testcase classname="performimg menu actions" name="vrifying ACCRON DISPLAYED test  scenario" time="0">
   <skipped message="Temporarily disabled with xit" />
  </testcase>
  <testcase classname="performimg menu actions" name="encountered a declaration exception" time="0">
   <skipped />
  </testcase>
 </testsuite>
 <testsuite name="verifyiong page titles" timestamp="2020-06-13T12:48:56" hostname="localhost" time="0" errors="0" tests="0" skipped="0" disabled="0" failures="0">
 </testsuite>
 <testsuite name="demo automation homepage demo" timestamp="2020-06-13T12:48:56" hostname="localhost" time="35.432" errors="0" tests="3" skipped="0" disabled="0" failures="0">
  <testcase classname="demo automation homepage demo" name="populating the register formurlCurrent_Content_FirstDataSets" time="12.49" />
  <testcase classname="demo automation homepage demo" name="populating the register formurlCurrent_Content_SecondDataSets" time="12.351" />
  <testcase classname="demo automation homepage demo" name="populating the register formurlCurrent_Content_ThirdDataSets" time="10.576" />
 </testsuite>
 <testsuite name="verifyiong page titles" timestamp="2020-06-13T12:49:32" hostname="localhost" time="0" errors="0" tests="0" skipped="0" disabled="0" failures="0">
 </testsuite>
 <testsuite name="verifyiong page titles" timestamp="2020-06-13T12:49:32" hostname="localhost" time="17.762" errors="0" tests="3" skipped="0" disabled="0" failures="0">
  <testcase classname="verifyiong page titles" name="vrifying opage title for subMenuPageTitle1" time="10.384" />
  <testcase classname="verifyiong page titles" name="vrifying opage title for subMenuPageTitle2" time="4.147" />
  <testcase classname="verifyiong page titles" name="vrifying opage title for subMenuPageTitle3" time="3.227" />
 </testsuite>
 <testsuite name="verifyiong page titles" timestamp="2020-06-13T12:49:49" hostname="localhost" time="0" errors="0" tests="0" skipped="0" disabled="0" failures="0">
 </testsuite>
</testsuites>